client/views/omnichannel/departments/EditDepartmentWithData.js

```diff
@@ -10,14 +10,16 @@ import EditDepartment from './EditDepartment';
 
 function EditDepartmentWithData({ id, reload, title }) {
 	const t = useTranslation();
-	const { value: data, phase: state, error } = useEndpointData(`livechat/department/${id}`);
+	const onlyMyDepartments = true;
+	const param = useMemo(() => ({ onlyMyDepartments }), [onlyMyDepartments]);
+	const { value: data, phase: state, error } = useEndpointData(`livechat/department/${id}`, param);
 
 	if ([state].includes(AsyncStatePhase.LOADING)) {
 		return <FormSkeleton />;
 	}
 
-	if (error) {
-		return <Box mbs='x16'>{t('User_not_found')}</Box>;
+	if (error || (id && !data?.department)) {
```

Comment:
```suggestion
	if (error || (!data || !data.department)) {
```

Reply:
We can't follow this suggestion, otherwise, the user will get `Department not found` every time they attempt to create a new department.